exchange/exchange-ps/exchange/mailboxes/Restore-RecoverableItems.md

```diff
@@ -101,7 +101,7 @@ After using the Get-RecoverableItems cmdlet to verify the existence of the item,
 ## PARAMETERS
 
 ### -Identity
-The Identity parameter specifies the mailbox that contains the Recoverable Items folder where you want to restore items from. You can use any value that uniquely identifies the mailbox. For example:
+The Identity parameter specifies the list of mailbox that contains the Recoverable Items folder where you want to restore items from. You can use any value that uniquely identifies the mailbox. For example:
```

Comment:
The Identity parameter specifies the list of mailboxes that contain the Recoverable Items folder that you want to restore items from. You can use any value that uniquely identify mailboxes. For example: